test/extended/builds/pipeline.go

```diff
@@ -228,6 +228,32 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 		})
 	})
 
+	g.Context("Pipeline log annotation check", func() {
+		g.AfterEach(func() {
+			if os.Getenv(jenkins.DisableJenkinsGCStats) == "" {
+				g.By("stopping jenkins gc tracking")
+				ticker.Stop()
+			}
+		})
+
+		g.It("should build and complete successfully", func() {
+			// instantiate the bc
+			g.By(fmt.Sprintf("calling oc new-app -f %q", envVarsPipelinePath))
+			err := oc.Run("new-app").Args("-f", envVarsPipelinePath).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// start the build
+			g.By("starting the pipeline build")
+			br, _ := exutil.StartBuildAndWait(oc, "sample-pipeline-withenvs")
+			// note, this will check the annotations if there is a build failure
+			debugAnyJenkinsFailure(br, oc.Namespace()+"-sample-pipeline-withenvs", oc, true)
+			br.AssertSuccess()
+
+			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
```

Comment:
can't we just check for these in one of the other tests that's already running a pipeline build?